vercel-functions/prerender/campus/[institution_slug].func/.vc-config.json

```diff
@@ -0,0 +1,9 @@
+{
```

Comment:
@Arpan-206 this whole folder needs to be a symlink - example: 

<img width="793" height="144" alt="Screenshot 2025-08-14 at 20 22 51" src="https://github.com/user-attachments/assets/9b8b2d91-cbd9-429c-96c6-aaa92c471fa8" />


Only the `prerender-config.json` file needs to be maintained separately.